### DISCUSSION TITLE
Cannot change the parameter RS5

### DISCUSSION THREAD

**Discussion by yfc26** [Yaskawa-Global/YMConnect#21] [Q&A] [answered]
I'm following Quick start -Robot Controller Setup for DX200, all the other RS parameters can be changed ,but when I change the RS5, after entering the value, it will change back to 0. Anyone know how to deal with this situation?

**Comment by ted-miller** [comment]
Hi @yfc26 

I believe this behavior will occur if there is another conflicting option enabled on the robot controller.

Please boot into Maintenance mode and save a CMOS.BIN backup of your controller. Then zip that file and attach it here. We will work to determine what the conflict is.

**Comment by yfc26** [comment]
Hi ted-miller,
please find the attached CMOS.BIN. Thank you very much for you help.
[CMOS.zip](https://github.com/user-attachments/files/16791716/CMOS.zip)

**Reply by ted-miller** [reply, marked as answer]
@yfc26 
The Network function is not enabled on your robot controller. So the ethernet adapter is not active.

Please contact your local Yaskawa office for assistance with enabling this function.
https://www.motoman.com/en-us/about/company/locations

**Reply by yfc26** [reply]
I see. Thank you for your help.
